util.py

```diff
@@ -0,0 +1,12 @@
+def convert_str_to_bool(value: str, default: bool) -> bool:
```

Comment:
<div>

<strong>Bito Code Review Agent Run</strong> <a href=https://github.com/rkj6124/test/pull/2#issuecomment-2399609989>#b5d3aa</a><span> - 10/08/2024, 05:15 pm</span>
<br>

<div><small>🟠</small> Medium importance</div>

<div id="suggestion">
<div id="issue">
<b>Issue:</b> The type hint for the 'value' parameter should be Optional[str] instead of str, as the function explicitly handles None values.
</div>
<div id="fix">
<b>Fix:</b> Change the type hint of the 'value' parameter to Optional[str]
</div>
<details>
<summary>
<b>Code suggestion</b>
</summary>
<div id="code">

```
 @@ -1 +1 @@
 -def convert_str_to_bool(value: str, default: bool) -> bool:
 +def convert_str_to_bool(value: Optional[str], default: bool) -> bool:
```
</div>
</details>
</div>
<br>


</div>

Is this a valid issue, or was it incorrectly flagged by the Agent?
- [ ] it was incorrectly flagged